src/InputWithLabel.jsx

```diff
@@ -4,7 +4,7 @@ const InputWithLabel = ({ todoTitle, handleTitleChange, children }) => {
   const inputRef = useRef();
   useEffect(() => {
     inputRef.current.focus();
-  });
+  }, []);
```

Comment:
Instead of the empty array leave it as blank. It should be fine for this specific case since it's a small app

Reply:
Fixed.